src/native/s3select/s3select_napi.cpp

```diff
@@ -0,0 +1,230 @@
+/* Copyright (C) 2016 NooBaa */
+
+#include "../../../submodules/s3select/include/s3select.h"
+#include "../util/common.h"
+#include "../util/napi.h"
+
+namespace noobaa {
+
+DBG_INIT(0);
+
+enum InputFormat {csv, json, parquet};
+
+class SelectWorker : public Napi::AsyncWorker
+{
+public:
+	Napi::Promise::Deferred *deferred = NULL;
```

Comment:
Why `NULL` and not `nullptr`?
(I comment on this line, but it is in general in cpp files).